src/components/ui/Text.tsx

```diff
@@ -352,13 +353,15 @@ export function CardSubheader({ children }: ChildrenProp) {
   return <p className={cardSubheaderContainer}>{children}</p>
 }
 
+const subHeader = (defaultFont?: boolean, textSize?: TFontSize) =>
```

Comment:
We decided not to use generics like `TFontSize`, so you need to add a `small ? 'text-sm' : ...` prop for example, because if you have any text size, what is the point of having this component?